### PR TITLE
Unpend and update teams section specs [PEOPLE-101]

### PR DESCRIPTION
**Pull request by smnkrt**
**JIRA:** https://netguru.atlassian.net/browse/PEOPLE-101

**Changes:**
 - add validation
 - update class names
 - remove unused class
 - unpend and update specs

### CODE REVIEW

File: spec/features/teams_spec.rb [hunk 1]
```diff
@@ -104,60 +123,74 @@
...
+
+    let(:error_msg) do
+      'New name not provided'
+    end
+
+    it 'shows edit form' do
+      expect(page).to have_css('.modal-dialog.edit-team')
     end
 
+    # TODO find out why failing on circle
```
Comment (netguru-hound) [inline spec/features/teams_spec.rb]: Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem.

File: app/react/components/teams/team.jsx [hunk 2]
```diff
@@ -49,6 +49,21 @@ class Team extends React.Component {
     return items;
   }
 
+  visibleRoles() {
+    return this.props.roles.filter(role => {
+      return role.show_in_team === true
```
Comment (netes) [inline app/react/components/teams/team.jsx]: Missing semicolon at the end

File: app/react/components/teams/team.jsx [hunk 3]
```diff
@@ -49,6 +49,21 @@ class Team extends React.Component {
...
+    });
+  }
+
+  visibleUsers() {
+    let visibleRolesIds = this.visibleRoles().map((role) => role.id);
```
Comment (netes) [inline app/react/components/teams/team.jsx]: Should be `const`, you're not changing this variable later.

**Comment by netes** [comment]
Besides that two comments - everything's ok :+1:

**Pull request merged**